content/en/docs/reference/glossary/api-resource.md

```diff
@@ -0,0 +1,18 @@
+---
+title: API resource
+id: api-resource
+date: 2025-02-09
+full_link: /docs/reference/using-api/api-concepts/#standard-api-terminology
+short_description: >
+  A Kubernetes entity, representing an endpoint on the Kubernetes API server.
+
+aka:
+tags:
+- fundamental
+---
+An entity in the Kubernetes type system, corresponding to an API endpoint on the {{< glossary_tooltip text="Kubernetes API" term_id="kubernetes-api" >}}.
+A resource typically represents an {{< glossary_tooltip text="objects" term_id="object" >}}.
+But sometime represents an operation on objects, such as a permission check.
+<!--more-->
+API resources compromise the Kubernetes type system which makes up the majority of the Kubernetes API.
+Each resource represents an endpoint on the Kubernetes API server, defining the schema for the objects or operations on that resource.
```

Comment:
Would be great to mention that these are [resources](https://datatracker.ietf.org/doc/html/rfc7231#section-2) in the HTTP sense.

Reply:
good point, I've added in HTTP and URI terms